src/plugin/scoring-similarity/build.xml

```diff
@@ -18,7 +18,15 @@
 <project name="scoring-similarity" default="jar-core">
 
   <import file="../build-plugin.xml"/>
-
+	  <target name="deps-jar">
+	    <ant target="jar" inheritall="false" dir="../indexer-elastic" />
+	  </target>
+	  <!-- Add compilation dependencies to classpath -->
+	  <path id="plugin.deps">
+	    <fileset dir="${nutch.root}/build">
+	      <include name="**/indexer-elastic/*.jar" />
```

Comment:
In order to avoid adding lucene-core to the main ivy.xml and creating classpath version conflicts, I used the one in indexer-elastic. 
Is there a better way to do this, as somehow adding it to plugins ivy file is giving runtime errors or maybe I am missing something ?